starknet-foundry/crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -163,10 +248,21 @@ struct ScarbStarknetContractArtifact {
     casm: Option<PathBuf>,
 }
 
+fn convert_to_blockifier_felt(val: &Felt252) -> blockifier_Felt252 {
+    let v = val.to_bigint();
+    blockifier_Felt252::from(v)
+}
+
+fn convert_from_blockifier_felt(val: &blockifier_Felt252) -> Felt252 {
+    let v = val.to_bigint();
+    Felt252::from(v)
+}
```

Comment:
This could use some unit tests

Reply:
[Add tests for felt conversions](https://github.com/foundry-rs/starknet-foundry/pull/181/commits/ed8bb5f0ccd6217db212addc65080daddeb04dcd)